### PR TITLE
Tracker Docs update

### PR DESCRIPTION
**Pull request by gokulavasan**
Update docs to reflect TMS migration

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -68,36 +68,33 @@ Step 3: Create a CDAP application using the configuration file::
 
 Application Configuration File
 ------------------------------
-Create an application configuration file that contains the Kafka Audit Log reader configuration (the property
-``auditLogKafkaConfig``). It is the Kafka Consumer Flowlet configuration information.
+Create an application configuration file that contains the Zookeeper Quorum (not required in Standalone mode).
```
Comment (awholegunch) [inline README.rst]: ZooKeeper

File: README.rst [hunk 2]
```diff
@@ -68,36 +68,33 @@ Step 3: Create a CDAP application using the configuration file::
...
 
 This key contains a property map with:
 
 Required Properties:
 
-- ``zookeeperString``: Kafka Zookeeper string that can be used to subscribe to the CDAP audit log updates
-- ``brokerString``: Kafka Broker string to which CDAP audit log data is published
-
-*Note:* Specify either the ``zookeeperString`` or the ``brokerString``.
+- ``zookeeperString``: Zookeeper Quorum string that is used by CDAP [required only in Distributed mode]
```
Comment (awholegunch) [inline README.rst]: ZooKeeper quorum string that is used by CDAP (required only in CDAP Distributed mode)

File: README.rst [hunk 3]
```diff
@@ -34,13 +34,13 @@ Getting Started
...
-To use the 0.2.0 version of Tracker, you must have CDAP version 3.4.x or higher.
+To use the 0.3.0 version of Tracker, you must have CDAP version 4.0.x or higher.
 
 Audit Publishing to Kafka
 -------------------------
-The Tracker App contains a flow that subscribes to the Kafka topic to which CDAP publishes
+The Tracker App contains a flow that subscribes to the TMS topic to which CDAP publishes
 the audit updates. Before using this application, you should enable publishing of audit updates to
-Kafka by setting the ``audit.enabled`` option in your cdap-site.xml to ``true``.
+TMS by setting the ``audit.enabled`` option in your cdap-site.xml to ``true``.
```
Comment (awholegunch) [inline README.rst]: Could say something like:
```
TMS (Transactional Messaging Service, part of CDAP) ...
```
assuming that is what the acronym spells out to...

**Comment by awholegunch** [comment]
LGTM. Please check my changes.

**Comment by gokulavasan** [comment]
LGTM 👍

**Pull request merged**